id_1000_random/creating_tasks/2_make_bigquery_table.py

```diff
@@ -0,0 +1,159 @@
+"""
+The following script creates a bigtable table with all the tracks of vessels in a
+csv file to_classify_ + [string of the date the file was created]
+
+You should only change the variable today_date
+
+The code for downloading a table from BigQuery was writtne by Tim Hochberg:
+https://github.com/GlobalFishingWatch/nn-vessel-classification/blob/master/tah-proto/get-data/gctools.py
+"""
```

Comment:
Why  not just install that repo and import it instead of copying it? Then you get bug  fixes and your code is simpler.